src/kata6.joinNames.js

```diff
@@ -1,3 +1,10 @@
-const joinNames = namesObj => {};
+const joinNames = namesObj => {
+   const names = namesObj.map(nameObject => nameObject.name ) 
+    if(names.length > 1) {
+      const lastName = names.pop()
+     return (names.join(', ')+' & '+lastName)
```

Comment:
I think it makes the code more readable if you put spaces before and after the +